src/DoctrineBatchUtils/BatchProcessing/SimpleBatchIteratorAggregate.php

```diff
@@ -157,7 +165,7 @@ private function flushAndClearBatch($iteration)
     private function flushAndClearEntityManager()
     {
         $this->entityManager->flush();
-        $this->entityManager->clear();
+        $this->entityManager->clear($this->objectClass);
```

Comment:
This is not to be used: it's deprecated in `doctrine/orm` and is already gone from `develop` there, because clearing a subset of entities introduces horrible broken in-memory graph bugs.